app/views/CaptureInternationalAddress.scala.html

```diff
@@ -111,7 +103,40 @@
             defaultOptionText = messages("internationalAddress.country.default")
         )
 
+        @postcodeBlock
+
         @button(messages("app.common.continue"), showSaveProgressButton = true)
     }
 
+}
+
+@postcodeBlock = @{
+    if (isPpob) ppobBlock
+    else nonPpobBlock
+}
+
+@ppobBlock = {
+    @inputTextHintNonBold(
```

Comment:
the way this component displays the heading and the hint is very similar, did we check with design if this is how it's meant to be or do we need to change something else.

Reply:
May need to be a change - I'm not sure